meta-freescale-3rdparty-mbl/recipes-kernel/linux/linux-fslc_%.bbappend

```diff
@@ -30,28 +32,9 @@ SRC_URI_append_imx6ul-pico-mbl = " file://imx6ul-disable-cpu-idle-mbl.cfg"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=d7810fab7487fb0aad327b76f1be7cd7"
 
-do_preconfigure() {
-	mkdir -p ${B}
-	echo "" > ${B}/.config
-	CONF_SED_SCRIPT=""
-
-	kernel_conf_variable LOCALVERSION "\"${LOCALVERSION}\""
-	kernel_conf_variable LOCALVERSION_AUTO y
-
-	sed -e "${CONF_SED_SCRIPT}" < '${S}/arch/arm/configs/${KBUILD_DEFCONFIG}' >> '${B}/.config'
-
-	cfgs=`find ${WORKDIR}/ -maxdepth 1 -name '*-mbl.cfg' | wc -l`;
-	if [ ${cfgs} -gt 0 ]; then
-		${S}/scripts/kconfig/merge_config.sh -m -O ${B} ${B}/.config ${WORKDIR}/*-mbl.cfg
-	fi
-
-	if [ "${SCMVERSION}" = "y" ]; then
-		# Add GIT revision to the local version
-		head=`git --git-dir=${S}/.git rev-parse --verify --short HEAD 2> /dev/null`
-		printf "%s%s" +g $head > ${S}/.scmversion
-	fi
+do_preconfigure_prepend() {
+	cp ${S}/arch/arm/configs/${KBUILD_DEFCONFIG} ${WORKDIR}/defconfig
```

Comment:
We need this because the `do_preconfigure` task defined in `fsl-kernel-localversion.bbclass` looks for the config at `${WORKDIR}/defconfig` when it creates the `.config` file that our config fragments will be merged into. We need to copy `KBUILD_DEFCONFIG` there so it's picked up. 

We didn't need this before here because we were overriding the `do_preconfigure` task, copying and pasting most of the code and changing `${WORKDIR}/defconfig` to `'${S}/arch/arm/configs/${KBUILD_DEFCONFIG}'` (and adding the config fragment merging which is now in `mbl-kernel-config.bbclass`).

Reply:
Can you please add a comment explaining this?